### PR TITLE
Extends approaching roads in intersection template

### LINKED ISSUE #45: Map Request: intersection with stop lines and extended approach and exit
This is a second interation of #24. Here are the details:

intersection_stopline_extended_entry_exit

Note the 300m of approach / exit stretches of straight road in all four directions.
We can name this "**intersection_[number]m_width_[number]m_radius_stopline**".

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

Closes #45 

## Summary
 - Adds option for extending approaching road length.
 - Adds new resource:
   - resources/intersection/intersection_3_3m_width_6m_radius_300m_approach_stopline.xodr
   - image

## Test it
```
EXTENSIONS_LENGTH=300 WIDTH=3.3 RADIUS=6 CROSSWALK=False STOPLINE=True empy3 templates/intersection.xml.em > generated_file.xodr
```

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

### CODE REVIEW

**Comment by francocipollone** [comment]
Just noticed that the issue says:
```
We can name this "intersection_[number]m_width_[number]m_radius_stopline".
```

I named it `intersection_3_3m_width_6m_radius_300m_approach_stopline.xodr`

I can adapt names in a follow-up pr if you want. If so let me know if you want to keep the previous version of the map (that has 100 m of extension)

CC: @liangfok

**Comment by liangfok** [comment]
Can you add an intersection_3_3m_width_6m_radius_300m_approach_stopline.geojson?

File: resources/intersection/intersection_3_3m_width_6m_radius_300m_approach_stopline.xodr [hunk 1]
```diff
@@ -0,0 +1,702 @@
...
+            <laneLink from="1" to="1"/>
+            <laneLink from="-1" to="-1"/>
+        </connection>
+        <connection id="5" incomingRoad="3" connectingRoad="10" contactPoint="start">
+            <laneLink from="1" to="1"/>
+            <laneLink from="-1" to="-1"/>
+        </connection>
+    </junction>
+</OpenDRIVE>
+<!-- GeoJSON information
```
Comment (liangfok) [inline resources/intersection/intersection_3_3m_width_6m_radius_300m_approach_stopline.xodr:585]: Oh, I just noticed this GeoJSON at the bottom of the XODR file. Do we just copy this into its own .geojson file?

Comment (francocipollone) [reply resources/intersection/intersection_3_3m_width_6m_radius_300m_approach_stopline.xodr:585]: Yes it is a copy of this section. I run a script to extract the geojson file out of this.

**Comment by francocipollone** [comment]
> Can you add an intersection_3_3m_width_6m_radius_300m_approach_stopline.geojson?
> 
> It would be good to have that even if it is the same as https://github.com/maliput/maliput_xodr/blob/main/resources/intersection/intersection_3_3m_width_6m_radius_stopline.geojson?short_path=8a8cced, just so we have a 1:1 relationship between an .xodr and .geojson.

Oh I missed adding the geojson file yeah. I will add it asap, if you need it now for testing you can grab the geojson from the end of the xodr file.

**Comment by liangfok** [comment]
Thank you! Testing is ongoing.

**Review by liangfok — APPROVED** [review]
LGTM. This map passed downstream testing. Thank you.

**Pull request merged**